src/lib/Layout/SideBarLayout/SideBarLayoutContext.tsx

```diff
@@ -1,30 +0,0 @@
-import { createContext, PropsWithChildren, ReactNode, useContext } from "react";
```

Comment:
@severinoAmmirati  Can you check if it's used by old menu?

Reply:
@manni497 Is used by `SidebarLayout` and `SidebarLayoutContent` and its already replaced by new one